packages/flutter_tools/lib/src/commands/run.dart

```diff
@@ -102,6 +102,10 @@ class RunCommand extends RunCommandBase {
         help: 'Enable tracing of Skia code. This is useful when debugging '
               'the GPU thread. By default, Flutter will not log skia code.',
       )
+      ..addOption('trace-whitelist',
+        help: 'Filters out all trace events except those that are specified in '
```

Comment:
Add a `valueHelp` with some examples of parameters that could be passed

Reply:
done